ips_list.yml

```diff
@@ -72,7 +72,7 @@ riscv:
   commit: pulpissimo-v3.4.0
   domain: [cluster, soc]
 ibex:
-  commit: e65a27e14dda7bb3b09cadc4e9232305fd451cec
+  commit: 95b85ddd1c995ace9f89ee42530f9e24820c1051 # pulpissimo-v6.1.0 with bugfix in src_files
```

Comment:
Let me add another tag `pulpissimo-v6.1.1` for this. Otherwise, if we rebase the branch in the future, we might run into problems because this commit will be lost.

Reply:
Thanks, updated